### ISSUE TITLE
Missing focus management for overlay dismissal 

### ISSUE THREAD

**Issue by meganvilla** [OpenSpace/OpenSpace#3116]
When the "Scene" overlay is dismissed, by activating the "Close" ("x") button, the focus indicator is no longer visible, while screen reader users encounter complete silence in the speech output. 

**Suggested Solution** 
Manage keyboard focus to ensure that when the overlay is dismissed, focus reliably returns to the control that was used to open it. 

Severity: 1
